### PR TITLE
Format Telegram messages

### LINKED ISSUE #21: Format Telegram messages
Currently, Emulsion message in Telegram looks like that:

> ```
> <xmpp-user> text text
> ```

It should look like that instead:

> **xmpp-user** (← bold here)
> text text

[chat-linker](https://github.com/jt3k/chat-linker) does the same by default.

### PR DESCRIPTION
**Pull request by ForNeVeR**
Closes #21.

### CODE REVIEW

File: Emulsion/Telegram/Module.fs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module Emulsion.Telegram.Module
+
+open Emulsion
+open Emulsion.Settings
+
+type TelegramModule =
+    { run : TelegramSettings -> (Emulsion.Message -> unit) -> unit
```
Comment (gsomix) [inline Emulsion/Telegram/Module.fs]: I'm a bit late but it seems we can inject `TelegramSettings` using construction function.

```fsharp
type TelegramModule =
    { run  : Emulsion.Message -> unit -> unit
      send : OutgoingMessage  -> unit }

let create settings : TelegramModule =
    { run  = TelegramClient.run settings
      send = TelegramClient.send settings }
```

File: Emulsion/Telegram/Module.fs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module Emulsion.Telegram.Module
+
+open Emulsion
+open Emulsion.Settings
+
+type TelegramModule =
```
Comment (gsomix) [inline Emulsion/Telegram/Module.fs]: I'm not happy with naming. I'd suggest 

`Emulsion.Telegram.Module.TelegramModule -> Emulsion.Telegram.Client` 

and 

`Emulsion.Telegram.TelegramClient -> Emulsion.Telegram.Funogram`

File: Emulsion/Telegram/Html.fs [hunk 3]
```diff
@@ -0,0 +1,14 @@
+module Emulsion.Telegram.Html
+open System.Text
+
+/// HTML escaping for Telegram only. According to the docs: https://core.telegram.org/bots/api#html-style
+let escape (text : string) : string =
+    let result = StringBuilder text.Length
+    for char in text do
```
Comment (gsomix) [inline Emulsion/Telegram/Html.fs]: https://msdn.microsoft.com/visualfsharpdocs/conceptual/string.collect-function-%5bfsharp%5d

**Comment by ForNeVeR** [comment]
Thanks for your review! All of the concerns have been addressed.

**Pull request merged**